src/inputfield.js

```diff
@@ -50,17 +74,20 @@ class InputField extends Component {
     }
 
     render() {
-        const {onInputChange, ...restProps} = this.props
-
+        const otherProps = omit(this.props, [
```

Comment:
I don't quite understand why we need `omit.js` here. We can use [object spread](https://github.com/tc39/proposal-object-rest-spread) to implement the same thing. 
Besides, I want to make the dependencies as simple as possible.